docs/pages/versions/unversioned/sdk/localization.md

```diff
@@ -64,53 +67,87 @@ i18n.locale = Localization.locale;
 // When a value is missing from a language it'll fallback to another language with the key present.
 i18n.fallbacks = true;
 
-function App() {
+export default App => {
   return (
-    <Text>
-      {i18n.t('welcome')} {i18n.t('name')}
-    </Text>
+    <View style={styles.container}>
+      <Text style={styles.text}>
+        {i18n.t('welcome')} {i18n.t('name')}
+      </Text>
+    </View>
   );
-}
+};
+
+/* @hide const styles = StyleSheet.create({ ... }); */
+const styles = StyleSheet.create({
+  container: {
+    backgroundColor: '#fff',
+    alignItems: 'center',
+    justifyContent: 'center',
+    flex: 1,
+  },
+  text: {
+    fontSize: 20,
+  },
+});
+/* @end */
 ```
 
+</SnackInline>
+
 ## API
 
 ```ts
 import * as Localization from 'expo-localization';
 ```
 
-## Constants
-
 ### Behavior
 
 This API is mostly synchronous and driven by constants. On iOS the constants will always be correct, on Android you should check if the locale has updated using `AppState` and `Localization.getLocalizationAsync()`. Initially the constants will be correct on both platforms, but on Android a user can change the language and return, more on this later.
 
+## Constants
+
 ### `Localization.locale`
 
-Native device language, returned in standard format. Ex: `en`, `en-US`, `es-US`.
+An [IETF BCP 47 language tag](https://en.wikipedia.org/wiki/IETF_language_tag), consisting of a 2 character language-code and optional script, region and variant codes. Ex: `en`, `en-US`, `zh-Hans`, `zh-Hans-CN`, `en-emodeng`.
```

Comment:
```suggestion
An [IETF BCP 47 language tag](https://en.wikipedia.org/wiki/IETF_language_tag), consisting of a two-character language code and optional script, region and variant codes. Ex: `en`, `en-US`, `zh-Hans`, `zh-Hans-CN`, `en-emodeng`.
```

Also, I commented on this later but languages and locales are different and the ISO formats for them are subtly different as well on iOS. We need to be really careful and intentional about designing a universal API that makes sense across all platforms and ideally follows established ISO/IETF/etc... standards.

Reply:
It looks like W3C and other standards tend to use hyphens. The W3C unicode locale draft says all locale identifies are valid language tags, even. So I think what you have in this PR (namely, using BCP 47) is correct.

> Unicode Locale Identifier or Unicode Locale. A language tag that follows the additional rules and restrictions on subtag choice defined in UTR#35 [LDML]. Any valid Unicode locale identifier is also a valid [BCP47] language tag, but a few valid language tags are not also valid Unicode locale identifiers.